src/test_l1_plat_power.c

```diff
@@ -489,21 +486,21 @@ void test_l1_plat_power_positive_PLAT_API_SetWakeupSrc (void)
     char keyvalue[POWER_MANAGER_KEY_SIZE] = {0};
     uint8_t countOfWakeupSources = 0;
 
-    // Variation/Step 01: Call PLAT_INIT() - open interface
+    // Step 01: Call PLAT_INIT() - open interface
     ret = PLAT_INIT();
-    UT_ASSERT_EQUAL_FATAL(ret, PWRMGR_SUCCESS); // Ensure the returned value is PWRMGR_SUCCESS
+    UT_ASSERT_EQUAL_FATAL(ret, PWRMGR_SUCCESS);
 
-    // Variation/Step 02:Loop through all wakeup sources and set to false
-    countOfWakeupSources = UT_KVP_PROFILE_GET_UINT32("powermanager/CountOfWakeupSources");
+    // Step 02:Loop through all wakeup sources and set to false
+    countOfWakeupSources = UT_KVP_PROFILE_GET_UINT8("powermanager/CountOfWakeupSources");
     UT_LOG_DEBUG ("Count Of Wakeup Sources %d" , countOfWakeupSources);
-    
+
     for(int i = 0; i < countOfWakeupSources; i++){
         snprintf(keyvalue, POWER_MANAGER_KEY_SIZE, "powermanager/WakeupSources/%d", i);
-        srcType = UT_KVP_PROFILE_GET_UINT32(keyvalue);
+        srcType =(PWRMgr_PowerState_t)UT_KVP_PROFILE_GET_UINT32(keyvalue);
```

Comment:
this should be of type PWRMGR_WakeupSrcType_t ! please check everywhere correctly

Reply:
Done